app/views/users/post.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a full description of post <%= @post_id %> for User ID: <%= @user_id %></h1>
```

Comment:
- [ ] This file should not be under the `views/users` folder. It should be under the `views/posts` folder, and the file name should be `views/posts/show.html.erb` to match the posts show action.